### PR TITLE
[DOC-3304] Delegate 79707 release notes

### PR DESCRIPTION
**Pull request by brian-f-harness**
Delegate 79707 release notes

For reviewers, previews are available:

[What's new](https://649c2d08b4576204c3ac7fc7--harness-developer.netlify.app/release-notes/delegate)
[FirstGen Delegate release notes](https://649c2d08b4576204c3ac7fc7--harness-developer.netlify.app/docs/first-gen/firstgen-release-notes/fg-delegate)
[NextGen Delegate release notes](https://649c2d08b4576204c3ac7fc7--harness-developer.netlify.app/release-notes/delegate)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by pratmit — APPROVED** [review]

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 1]
```diff
@@ -5,12 +5,48 @@ These release notes document changes to Harness Delegate in Harness FirstGen.
...
+To identify the cluster that hosts your account, open Harness FirstGen, go to **Account Settings**, and then select **Overview**. The cluster is listed in **Harness Cluster Hosting Account**.
 :::
 
-For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md). For Self-Managed Enterprise Edition release notes, see [Self-Managed Enterprise Edition (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md).
+For FirstGen SaaS release notes, go to [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md). For Self-Managed Enterprise Edition release notes, go to [Self-Managed Enterprise Edition (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md).
 
-## Latest release - June 9, 2023, Harness version 79514, Harness Delegate version 79503
+
+## Latest - June 28, 2023, Harness version 797xx, Harness Delegate version 797xx
```
Comment (douglas-j-bothwell) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:14]: To-do: update version numbers

File: release-notes/delegate.md [hunk 2]
```diff
@@ -19,9 +19,9 @@ Harness Delegate, NextGen SaaS releases every two weeks. Harness Platform, NextG
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
 :::
 
-## Latest release - June 9, 2023, Harness version 79516, Harness Delegate version 79503
+## Latest - June 28, 2023, Harness version 797xx, Harness Delegate version 797xx
 
-Harness NextGen release 79516 includes the following changes for the Harness Delegate.
+Harness NextGen release 797xx includes the following changes for the Harness Delegate.
```
Comment (douglas-j-bothwell) [inline release-notes/delegate.md:24]: To-do: update release #

File: release-notes/delegate.md [hunk 3]
```diff
@@ -36,21 +36,100 @@ import Helmdep from '/release-notes/shared/helm-2-deprecation-notice.md'
...
+- You can now see disconnected delegate details in selection logs and error messages when there are no eligible delegates in an active state to execute tasks. (PL-37900)
+
+- Upgraded the delegate JRE to 11.0.19_7. (PL-37994)
+
+```mdx-code-block
+  </TabItem>
+  <TabItem value="Early access">
+```
+
+This release includes the following early access features and enhancements:
```
Comment (douglas-j-bothwell) [inline release-notes/delegate.md:48]:
```suggestion
This release includes the following early access feature:
```

File: release-notes/delegate.md [hunk 4]
```diff
@@ -36,21 +36,100 @@ import Helmdep from '/release-notes/shared/helm-2-deprecation-notice.md'
...
+
+- Added a new field in the release history for Blue Green deployments to differentiate between environments. (CDS-69961)
+
+  Enable the feature flag, `CDS_BG_STAGE_SCALE_DOWN_STEP_NG` to leverage this feature. 
+
+  This is an enhancement to the Kubernetes Blue Green Stage Scale Down step. You can now scale down your last successful stage environment only if the primary resources exist. This enhancement helps you efficiently manage your resources, and prevent deleting the important resources.
```
Comment (douglas-j-bothwell) [inline release-notes/delegate.md:54]: Is this a Kubernetes Blue Green Stage Scale Down step, or a Scale Down step in a Blue Green stage?

File: release-notes/delegate.md [hunk 5]
```diff
@@ -36,21 +36,100 @@ import Helmdep from '/release-notes/shared/helm-2-deprecation-notice.md'
...
+
+  When a multi-line string was passed as input for a child pipeline, the string was being converted to a single line. 
+
+  This issue is fixed. Instead of passing data using YAML, Harness now uses JSON for data processing. This helps preserve multi-line strings and YAML structures properly to process pipeline YAML and user inputs. Enable the feature flag, `PIE_PROCESS_ON_JSON_NODE` to leverage this fix.
+
+- Fixed an issue where the expression, `<+lastPublished.tag>.regex()` was not resolved properly when used as runtime input for artifacts. (CDS-68810)
+
+- Quotations were added to execution YAML strings inconsistently when comparing pipeline YAMLs. (CDS-67637)
+
+  This issue is fixed by enabling `MINIMIZE_QUOTES` for YamlUtils and YamlPipelineUtils classes. The compiled YAML no longer have quotations around strings where they are not needed, but only around numbers. Even if you had added quotations in the string values in the pipeline YAML, they'll be removed in the compiled YAML. Also, there won't be unnecessary audit trails where the diff only has quotations around strings.
```
Comment (douglas-j-bothwell) [inline release-notes/delegate.md:94]:
```suggestion
  This issue is fixed by enabling `MINIMIZE_QUOTES` for YamlUtils and YamlPipelineUtils classes. The compiled YAML no longer has quotations around strings where they are not needed, but only around numbers. Even if you had added quotations in the string values in the pipeline YAML, they'll be removed in the compiled YAML. Also, there won't be unnecessary audit trails where the diff only has quotations around strings.
```

**Review by douglas-j-bothwell — APPROVED** [review]
A few minor sugs but otherwise looks good

File: release-notes/delegate.md (hunk 3 above)
Comment (brian-f-harness) [reply release-notes/delegate.md:48]: Updated

File: release-notes/delegate.md (hunk 4 above)
Comment (brian-f-harness) [reply release-notes/delegate.md:54]: @douglas-j-bothwell This content is from the CD release notes. I'll check with @roshnisarangadharan.

File: release-notes/delegate.md (hunk 5 above)
Comment (brian-f-harness) [reply release-notes/delegate.md:94]: Updated

**Pull request merged**